spec/index.html

```diff
@@ -8849,16 +8849,27 @@ <h4>Variable Scope</h4>
                 <td><code>VALUES varlist { values }</code></td>
                 <td><code>v</code> is in-scope if <code>v</code> is in <code>varlist</code></td>
               </tr>
+              <tr>
+                <td>`EXISTS` and `NOT EXISTS` filters</td>
+                <td>
+                  <code>v</code> is in-scope if it is in-scope for the 
+                  <a href="#defn_sparqlSolutionMapping">solution mapping</a>
+                  where the `FILTER` containing `EXISTS` or `NOT EXISTS` is applied.
+                </td>
+              </tr>
             </tbody>
           </table>
-          <p>The variable <code>v</code> must not be in-scope at the point of the <code>(expr AS
-              v)</code> form. The scoping for <code>(expr AS v)</code> applies immediately in
-            <code>SELECT</code> expressions.</p>
+          <p>The variable <code>v</code> must not be in-scope at the point of the 
+            <code>(expr AS v)</code> form. The scoping for <code>(expr AS v)</code>
+            applies immediately in <code>SELECT</code> expressions.
+          </p>
           <p>In <code>BIND (expr AS v)</code> requires that the variable <code>v</code> is not
-            in-scope from the preceeding elements in the group graph pattern in which it is used.</p>
+            in-scope from the preceeding elements in the group graph pattern in which it is used.
+          </p>
           <p>In <code>SELECT</code>, the variable <code>v</code> must not be in-scope in the graph
             pattern of the <code>SELECT</code> clause, nor used in another select expression earlier in
```

Comment:
This appears to be non-normative text. I am therefore writing to replace the lowercase RFC2119 language.
```suggestion
          <p>In <code>SELECT</code>, the variable <code>v</code> cannot be in-scope in the graph
            pattern of the <code>SELECT</code> clause, nor have been used in another select expression earlier in
```